app/components/forms/wizard/attendee-step.js

```diff
@@ -2,15 +2,16 @@ import Component from '@ember/component';
 import { computed } from '@ember/object';
 import FormMixin from 'open-event-frontend/mixins/form';
 import EventWizardMixin from 'open-event-frontend/mixins/event-wizard';
-
+import { sortBy } from 'lodash-es';
 export default Component.extend(FormMixin, EventWizardMixin, {
 
   fixedFields: computed('data.customForms.@each', function() {
     return this.data.customForms?.filter(field => field.isFixed);
   }),
 
   editableFields: computed('data.customForms.@each', function() {
-    return this.data.customForms?.filter(field => !field.isFixed);
+    const fields = sortBy(this.data.customForms, ['isComplex', 'name']);
+    return fields?.filter(field => !field.isFixed);
```

Comment:
Filter and then sort. Also, should name be the second property to be used to sort. Is it like that since before?

Reply:
![scrnli_11_11_2020_6-41-32 PM](https://user-images.githubusercontent.com/56407566/98816152-3b8bd180-244e-11eb-921d-bdfcf0cbedf3.png)